tools/image_processing/imagej2/imagej2_convert_format/imagej2_convert_format.xml

```diff
@@ -36,24 +36,22 @@
     </outputs>
     <tests>
         <test>
-            <param name="input" value="dot_blot.jpg" ftype="jpg" />
-            <param name="max_heap_size_type" value="megabytes" />
-            <param name="max_heap_size" value="256" />
+            <param name="input" value="dot_blot.jpg" />
             <param name="input_datatype" value="jpg" />
             <param name="output_datatype" value="png" />
-            <output name="output" file="dot_blot.png" ftype="png" />
```

Comment:
@bgruening The tests pass without requiring the use of "sim_size", although the use of "lines_diff" is required.  I haven't tried replacing it with "sim_size", although I will if you'd like.  The problem is that using "type" in the tag results in errors like this:

Exception: {u'message': {u'type': u'error', u'data': {u'file_type': u"An invalid option was selected for file_type, u'jpg', please verify.", u'files_metadata': [u"An invalid option was selected for file_type, u'jpg', please verify."]}}}